testscommon/enableEpochsHandlerStub.go

```diff
@@ -1037,20 +1037,20 @@ func (stub *EnableEpochsHandlerStub) IsStakingV4Started() bool {
 	return stub.IsStakingV4StartedField
 }
 
-// StakingV4EnableEpoch -
-func (stub *EnableEpochsHandlerStub) StakingV4EnableEpoch() uint32 {
+// StakingV4Step2EnableEpoch -
```

Comment:
apply order here? Step1 first, second step2, and then step3?

Reply:
Refactored as suggested